src/Mono.Android/Android.Views/WindowManagerLayoutParams.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Android.Views
+{
+	partial class WindowManagerLayoutParams
+	{
+#if NET
```

Comment:
Given that we'll eventually do a `net7.0-android34.0` release, requiring cherry-picks to a release/7.0* branch, should this instead be:

```csharp
#if NET && ANDROID_34
```

Reply:
Guarded just in case.  However, I think we should expect `net7.0-android34.0` API may not exactly match `net8.0-android34.0` and we shouldn't backport these fixes.

I don't suspect a user upgrading to .NET 8, using new API, and then wanting to downgrade their app to .NET 7 is a common scenario.

Reply:
@jpobst: I'm more concerned about the opposite: customer *starts with* net7.0-android34.0 -- probably released in August or September! -- then migrating to net8.0-android34.0 in *November*.

Then there is ABI compatibility: libraries built against net7.0-android34.0 should work as-is in a net8.0-android34.0 app.  Will it?

Reply:
That should not be an issue with this PR, as it only adds API to newer versions, but it is good to keep that in mind in general.